sklearn/tests/test_naive_bayes.py

```diff
@@ -647,14 +659,65 @@ def test_cnb():
     assert_array_almost_equal(clf.feature_log_prob_, normed_weights)
 
 
-def test_naive_bayes_scale_invariance():
```

Comment:
Moved up to other gnb tests and renamed to `test_gnb_naive_bayes_scale_invariance`